framework/src/actions/CreateProblemAction.C

```diff
@@ -47,6 +47,15 @@ validParams<CreateProblemAction>()
                                        "<path>/<filebase> or <path>/LATEST to "
                                        "grab the latest file available)");
 
+#ifdef LIBMESH_HAVE_PETSC
+  params.addParam<MultiMooseEnum>(
+      "petsc_options", MultiMooseEnum("", "", true), "TODO: add doc message");
+  params.addParam<std::vector<std::string>>(
+      "petsc_inames", std::vector<std::string>(), "TODO: add doc message");
+  params.addParam<std::vector<std::string>>(
+      "petsc_values", std::vector<std::string>(), "TODO: add doc message");
+#endif
```

Comment:
Hmm, this isn't quite right. These parameters should be added through this method:
https://github.com/idaholab/moose/blob/devel/framework/src/utils/PetscSupport.C#L758-L803